user_sync/connector/umapi.py

```diff
@@ -318,7 +319,8 @@ def create_action(self, commands):
             action = umapi_client.UserAction(umapi_identity_type, email, username, domain,
                                              requestID=self.get_next_request_id())
         except ValueError as e:
-            raise AssertionException("Error creating umapi Action: %s" % e)
+            self.logger.error("Skipping user - Error creating umapi Action: %s" % e)
+            return
```

Comment:
You should explicitly return `None` from value-producing functions, as it helps with type analysis and automatic error checking.  Especially when you are returning class objects otherwise, as in this case you are using `None` specifically as a flag value.
```suggestion
            return None
```